dev-libs/libqt6pas/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>amynka@gentoo.org</email>
+        <name>Amy Liffey</name>
+    </maintainer>
+    <maintainer type="person" proxied="yes">
+        <email>voron1@gmail.com</email>
```

Comment:
Can you remove voron1 as maintainer, since he didnt agree to take this package..

Reply:
Should I just remove the XML-node?